platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/chatter/protocol/peer/PeerGossipState.java

```diff
@@ -81,19 +78,17 @@ public synchronized void shiftWindow(final long generation) {
     /**
      * Handle the descriptor received by this peer
      *
-     * @param descriptor
-     * 		the descriptor received
+     * @param descriptor the descriptor received
      */
-    public synchronized void handleDescriptor(final ChatterEventDescriptor descriptor) {
+    public synchronized void handleDescriptor(final EventDescriptor descriptor) {
```

Comment:
Null annotations and checks missing

Reply:
I am not going to add those here, since I only changed the type.